server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1521,9 +1521,11 @@ public UeberCertificate getUeberCertificate(@Context Principal principal,
         @PathParam("owner_key") @Verify(Owner.class) String ownerKey,
         @QueryParam("hypervisor_id") List<String> hypervisorIds) {
 
-        return (hypervisorIds == null || hypervisorIds.isEmpty()) ?
+        CandlepinQuery<Consumer> out = (hypervisorIds == null || hypervisorIds.isEmpty()) ?
             this.consumerCurator.getHypervisorsForOwner(ownerKey) :
             this.consumerCurator.getHypervisorsBulk(hypervisorIds, ownerKey);
```

Comment:
this is still using ownerKey even though the getHypervisorsBulk was changed to use the owner ID